### DISCUSSION TITLE
Text is dark in some apps in dark mode

### DISCUSSION THREAD

**Discussion by KevinNitroG** [prasanthrangan/hyprdots#1525] [Q&A]
Hi, I have some problems with dark mode. In some apps the text is dark even I'm using dark mode. I can't remember that I have customised sth. But I think the the problem came first when I adjusted sth related to wallbash. 

- Qt6 Setting
  > ![image](https://github.com/prasanthrangan/hyprdots/assets/86353526/4f0165e0-05f5-4b88-988a-7a331ebb40a4)
- fcitx setting
  > ![image](https://github.com/prasanthrangan/hyprdots/assets/86353526/0f9c3b48-f32d-4d61-be60-3eac11e3d7b9)

Thank you guys.

**Comment by kRHYME7** [comment]
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/51b254fd-c3db-4f82-a543-fffaf6005b6d)

For the meantime, if selecting this is readable?

**Reply by KevinNitroG** [reply]
It still doesn't work out :(

**Reply by kRHYME7** [reply]
Same color ? Can you select another theme ? 

cont... 

As I investigated, there are certain elements that are not covered by the theme and wallbash. 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/b4b0a53f-53e4-4df6-bcf8-fb6a13920bad)

All this qt6 setting windows  do not change that black text so we assume that it is not changed dynamically. 


If we could find this certain element, maybe we can fix this. Would be grateful to receive a PR.

**Reply by KevinNitroG** [reply]
Thank you. I'm looking forward to the fix 🌝 (or some PRs) (sorry I'm new to Linux and I can't help anything)

**Comment by decide-you** [comment]
@KevinNitroG update dotfiles to the latest.
